assets/chessground.brown.css

```diff
@@ -42,17 +42,21 @@ cg-board square.current-premove {
   background-color: rgba(20, 30, 85, 0.5);
 }
 
-/** Alternating colors in rank/file labels */
+/** Alternating colors in rank/file/square labels */
 .cg-wrap.orientation-white coords.ranks coord:nth-child(2n),
 .cg-wrap.orientation-white coords.files coord:nth-child(2n),
 .cg-wrap.orientation-black coords.ranks coord:nth-child(2n + 1),
-.cg-wrap.orientation-black coords.files coord:nth-child(2n + 1) {
+.cg-wrap.orientation-black coords.files coord:nth-child(2n + 1),
+.cg-wrap coords.squares:nth-child(2n + 2) coord:nth-child(2n + 1),
+.cg-wrap coords.squares:nth-child(2n + 1) coord:nth-child(2n) {
   color: rgba(72, 72, 72, 0.8);
 }
 
 .cg-wrap.orientation-black coords.ranks coord:nth-child(2n),
 .cg-wrap.orientation-black coords.files coord:nth-child(2n),
 .cg-wrap.orientation-white coords.ranks coord:nth-child(2n + 1),
-.cg-wrap.orientation-white coords.files coord:nth-child(2n + 1) {
+.cg-wrap.orientation-white coords.files coord:nth-child(2n + 1),
+.cg-wrap coords.squares:nth-child(2n + 1) coord:nth-child(2n + 1),
+.cg-wrap coords.squares:nth-child(2n + 2) coord:nth-child(2n) {
```

Comment:
are we sure we need 2x3 new selectors here? which would be multiplied by the number of board themes

Reply:
One can simplify all those selectors to the minimum, but it might break other themes?
```css
/** Alternating colors in rank/file/square labels */
.cg-wrap coords:nth-child(odd) coord:nth-child(even),
.cg-wrap coords.squares:nth-child(even) coord:nth-child(odd),
.cg-wrap coords.files:nth-child(even) coord:nth-child(even) {
  color: rgba(72, 72, 72, 0.8);
}

.cg-wrap coords:nth-child(odd) coord:nth-child(odd),
.cg-wrap coords.squares:nth-child(even) coord:nth-child(even),
.cg-wrap coords.files:nth-child(even) coord:nth-child(odd) {
  color: rgba(255, 255, 255, 0.8);
}

```